docs/building-images.md

```diff
@@ -198,6 +218,18 @@ kubernetes:
 * `manifests` - Optional; manifests to apply to the cluster.
   Can be used separately or in combination with the [configuration directory](#kubernetes-1).
   * `urls` - Optional; list of HTTP(s) URLs to download the manifests from
+* `helm` - Optional; Helm charts to be deployed to the cluster.
+  * `charts` - Optional; Defines a list of Helm charts and configuration for each Helm chart.
+    * `name` - Required; This must match the name of the actual Helm chart.
+    * `repositoryName` - Required; This is the name of the corresponding `name` for a repository/registry specified within `repositories` that contains this Helm chart.
+    * `version` - Required; The version of the Helm chart to be deployed.
+    * `installationNamespace` - Optional; The namespace where the Helm installation is executed. The default is `default`.
+    * `targetNamespace` - Optional; The namespace where the Helm chart will be deployed. The default is `default`.
+    * `createNamespace` - Optional; If `true` the `targetNamespace` will be created, if `false` it assumes the `targetNamespace` already exists.
```

Comment:
What happens if it assumes it exists but it doesn't? I'm guessing runtime error when the node is booted but might make sense to explicitly mention.

Reply:
Will add that 👍